src/config/themeConfig.js

```diff
@@ -23,82 +23,81 @@ import { primary as defaultPrimary } from "./themes/default/defaultTheme";
 import {
   primaryDarker as defaultPrimaryDarker,
   darkColor as defaultDarkColor,
-  headerColor as defaultBanner,
 } from "./themes/default/defaultTheme";
 
 export const getTiamatEnv = () => {
   return getIn(window, ["config", "tiamatEnv"], "development");
 };
 
 export const getEnvColor = (env) => {
-  if (process.env.REACT_APP_THEME) {
+  if (import.meta.env.VITE_THEME) {
     return require(
-      "./themes/" + process.env.REACT_APP_THEME + "/index.js",
+      "./themes/" + import.meta.env.VITE_THEME + "/index.js",
     ).getEnvColor(env);
   } else {
     return defaultEnvColor(env);
   }
 };
 
 export const getV0Theme = () => {
-  if (process.env.REACT_APP_THEME) {
+  if (import.meta.env.VITE_THEME) {
     return require(
-      "./themes/" + process.env.REACT_APP_THEME + "/index.js",
+      "./themes/" + import.meta.env.VITE_THEME + "/index.js",
     ).getV0Theme();
   } else {
     return getV0DefaultTheme();
   }
 };
 
 export const getTheme = () => {
-  if (process.env.REACT_APP_THEME) {
+  if (import.meta.env.VITE_THEME) {
     return require(
-      "./themes/" + process.env.REACT_APP_THEME + "/index.js",
+      "./themes/" + import.meta.env.VITE_THEME + "/index.js",
     ).getTheme();
   } else {
     return getDefaultTheme();
   }
 };
 
 export const getLogo = () => {
-  if (process.env.REACT_APP_THEME) {
-    return require("./themes/" + process.env.REACT_APP_THEME + "/logo.png")
+  if (import.meta.env.VITE_THEME) {
+    return require("./themes/" + import.meta.env.VITE_THEME + "/logo.png")
       .default;
   } else {
     return defaultLogo;
   }
 };
 
 export const getPrimaryColor = () => {
-  if (process.env.REACT_APP_THEME) {
-    return require("./themes/" + process.env.REACT_APP_THEME + "/index.js")
+  if (import.meta.env.VITE_THEME) {
+    return require("./themes/" + import.meta.env.VITE_THEME + "/index.js")
       .primary;
   } else {
     return defaultPrimary;
   }
 };
 
 export const getHeaderColor = () => {
-  if (process.env.REACT_APP_THEME) {
-    return require("./themes/" + process.env.REACT_APP_THEME + "/index.js")
+  if (import.meta.env.VITE_THEME) {
+    return require("./themes/" + import.meta.env.VITE_THEME + "/index.js")
       .primary;
   } else {
-    return headerColor;
+    return defaultEnvColor;
```

Comment:
This is where headerColor was used, I replaced it with defaultEnvColor, but this might be wrong.

Reply:
I think it is fine for now. The Theme set up works as is, but this should be changed in the future to support a more flexible customization.